src/index.ejs

```diff
@@ -22,6 +22,19 @@
   <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/named-exports.min.js"></script>
 </head>
 <body>
+  <single-spa-router>
+    <nav class="topnav">
```

Comment:
The [devtools selector](https://github.com/react-microfrontends/navbar/blob/a6839022281b557ce085a72ff4be128c133554c6/src/react-mf-navbar.js#L38) should be changed to match this as well probably. It wasn't working before anyway.

Reply:
Good catch - I will remove the devtools selector from the navbar app. Single-spa-layout uses `<div id="single-spa-application:name"></div>` for the application containers, which the devtools automatically pick up on.

In other words, this single-spa-layout config will result in the following DOM:

```html
<nav class="topnav">
  <div id="single-spa-application:@react-mf/navbar">
  </div>
</nav>
```

Reply:
Oh yea, that's even better!